_o_components/35-o-table.component.md

```diff
@@ -1139,6 +1139,75 @@ this.table.reinitialize({ columns: columnsOfTable, visibleColumns: columnsOfTabl
 ...
 
 ```
+
+### Expandable row <span class='menuitem-badge'>new<span>
+
+The `o-table-row-expandable` component enable you to provide additional details about a particular row of table data through expanding or collapsing its content. It's necessary wrap the content of your template with the `<ng-template let-row></ng-template` tag and add the template definitio inside.
+
+*Ontimize Web* provides two posibilities for attaching child rows to a parent row in the `o-table`. 
+* Simple template
+* A template with context
+
+**Simple template**
+
+The example below makes use of *simple template*, the content of the child row in this example is defined by the `ng-template` with the `let-row` attribute so that exposes the variable `row` in the template to show the information.
+
+```html
+ <o-table fxFill #table service-type="DummyService" service="customers" entity="customer" keys="CUSTOMERID" columns="CUSTOMERID;SURNAME;NAME"
+      title="CUSTOMERS" insert-button="no" delete-button="no" refresh-button="no" pagination-controls="yes" detail-mode="none" export-button="no"
+      query-rows="10" fixed-header="yes">
+      <o-table-row-expandable>
+        <ng-template let-row>
+          <o-column title="CONTACT_DATA" icon="info" class="vertical-margin-10" layout-gap="12px">
+            <div fxLayout="row wrap" fxLayoutGap="14px">
+              <span fxFlex="30%"><strong>{% raw %}{{'ADDRESS' | oTranslate}}{% endraw %}</strong>: {% raw %}{{row.ADDRESS}}{% endraw %}</span>
+              <span fxFlex="20%"><strong>{% raw %}{{'COUNTRY' | oTranslate}}{% endraw %}</strong>: {% raw %}{{row.COUNTRY}}{% endraw %}</span>
+              <span fxFlex="20%"><strong>{% raw %}{{'STATE' | oTranslate}}{% endraw %}</strong>: {% raw %}{{row.STATE}}{% endraw %}</span>
+              <span fxFlex="20%"><strong>{% raw %}{{'ZIPCODE' | oTranslate}}{% endraw %}</strong>: {% raw %}{{row.ZIPCODE}}{% endraw %}</span>
+            </div>
+            <div fxLayout="row wrap" fxLayoutGap="14px">
+              <span fxFlex="20%"><strong>{% raw %}{{'EMAIL' | oTranslate}}{% endraw %}</strong>: {% raw %}{{row.EMAIL}}{% endraw %}</span>
+              <span fxFlex="20%"><strong>{% raw %}{{'PHONE' | oTranslate}}{% endraw %}</strong>: {% raw %}{{row.PHONE}}{% endraw %}</span>
+            </div>
+          </o-column>
+        </ng-template>
+      </o-table-row-expandable>
+
+    </o-table>
+```
+
+![Row expanded]({{ "/images/components/tabla/row-expanded.png" | absolute_url }}){: .comp-example-img}
+
+**A template with context**
+
+It is possible to define a *template with context*, so that if the defined template contains a component that extends from `OServiceComponent` class (such as o-table, o-list, o-chart and o-grid), this context can solve functionalities like parent-keys and query the component data automatically when the row is expanded, for example.
```

Comment:
- ... component that extends from `OServiceComponent` class ... -> **from** not necessary
- ... this context can solve functionalities like ... -> this context **provide** functionalities